webapp/src/app/shared/widgets/event-widget/event-editor/editor/event-editor.component.ts

```diff
@@ -25,6 +29,7 @@ export class EventEditorComponent implements OnChanges {
 	
 	@Input() eventData: EventType[] = [];
 	@Input() actionStep = false;
+	@Input() wrapTextOverride?: boolean; //If assigned uses the assigned value, if undefined uses the value from the settings
```

Comment:
I would remove this currently because it's not used. If there is ever a use case for it it's not difficult to implement.